pkg/container-collection/container-collection.go

```diff
@@ -442,6 +442,7 @@ func (cc *ContainerCollection) EnrichByMntNs(event *eventtypes.CommonData, mount
 		event.Runtime.RuntimeName = container.Runtime.RuntimeName
 		event.Runtime.ContainerName = container.Runtime.ContainerName
 		event.Runtime.ContainerID = container.Runtime.ContainerID
+		event.Runtime.ImageName = container.Runtime.ImageName
```

Comment:
You're missing to do the same in `EnrichEventByMntNs` and `EnrichEventByNetNs`.

Reply:
I thought this is automatically done on [this line](https://github.com/claudiamarcubina/inspektor-gadget/blob/14233a54cf36cf253a4b2f5d6e9641297544d898/pkg/container-collection/operator.go#L30C3-L30C3) and on [this one](https://github.com/claudiamarcubina/inspektor-gadget/blob/14233a54cf36cf253a4b2f5d6e9641297544d898/pkg/container-collection/operator.go#L46), isn't it?

Reply:
You're right.